src/js/disability-benefits/actions/index.js

```diff
@@ -211,10 +209,12 @@ export function submitFiles(claimId, trackedItem, files) {
 
     files.forEach(({ file, docType }) => {
       uploader.addFiles(file, {
-        docType: docType.value,
-        docTypeDescription: getDocTypeDescription(docType.value)
+        tracked_item_id: trackedItemId,
+        document_type: docType.value,
+        document_type_description: getDocTypeDescription(docType.value)
```

Comment:
You don't need the description. It's added on the backend, just because it's redundant with the doc type.